AppBuilder/ABFactory.js

```diff
@@ -39,10 +39,10 @@ import Tenant from "../resources/Tenant.js";
 import UISettings from "./uiSettings/config.js";
 // UISettings: detailed settings for our common UI elements
 
-import Webix from "../js/webix/webix.min.js";
+// import Webix from "../js/webix/webix.min.js";
 // NOTE: moved to require() because using import with webix.js
 // really messed things up!
-// var Webix = require("../js/webix/webix.js");
+var Webix = require("../js/webix/webix.js");
```

Comment:
I don't understand why we would need to change this now?  We've been using import for about a year already.

Reply:
I'm sorry. I did not clean up the code. 

I uses it for debugging.